### ISSUE TITLE
Link ASIO setup page on error message if no ASIO driver was found

### ISSUE THREAD

**Issue by ann0see** [jamulussoftware/jamulus#1756]
**Has this feature been discussed and generally agreed?**
No. But it's a simple fix.
**Describe the solution you'd like**
If you start Jamulus without an ASIO driver, the following message greets new users:

![NoASIO](https://user-images.githubusercontent.com/20726856/119361113-97f86500-bcab-11eb-9b70-f60b2c492b95.png)

Although I made the text less verbose, it's still a lot to read. 
From my point of view, greeting a new user with an error message is bad UX. We should link the website for more information or think about something more friendly.

**Proposed solutions**

* Add a button which links to the website (needs improved exception handling probably)
* Add a clickable link in the error message (easy but not visible)
* Do some fancy stuff: run an onboarding wizard?
* Leave it as is since the installer already generates a warning

**Comment by henkdegroot** [comment]
I noticed when doing a build "with jack" there is also a message about missing ASIO, but at least that allows you to continue and show a different dialog after that with a hyper-link to where you should. (writing this from memory...not 100% sure about the dialog sequence). But perhaps that is something to consider for this dialog as well?

**Comment by ann0see** [comment]
That’s probably the installer?

**Comment by henkdegroot** [comment]
> That’s probably the installer?

You are correct....obvious I did not read the issue well enough....and it even says "If you start Jamulus".....guessing we could look at adding the very same button as we show in the installer dialog?

**Comment by ann0see** [comment]
Yes. It's a bit different since it's C++ not NSIS code.

**Comment by henkdegroot** [comment]
@gilgongo, so should we extend the "error" dialog and add a link to: https://jamulus.io/wiki/Installation-for-Windows  ?

**Comment by gilgongo** [comment]
I think so. We can probably anchor it to the ASIO section I think https://jamulus.io/wiki/Installation-for-Windows#asio

**Comment by henkdegroot** [comment]
Thought about that as well....a similar dialog is presented when using the JACK version.
Should we update that dialog as well with a similar reference? We could use the anchor jack for this...and if that section will become available, you won't need to update the software and if the anchor is not there it just opens the web page.

Can the change for the JACK dialog be covered in the same PR?
Do we need to create another issue for JACK? Or can we update this one to include the reference to JACK?

**Comment by gilgongo** [comment]
Ah yes, that would be good to cover JACK as well (so the link to https://jamulus.io/wiki/Installation-for-Linux#set-up-your-hardware perhaps?)

May as well include it in the same PR I think.

**Comment by henkdegroot** [comment]
I never checked, but does the dialog get generated for Linux/macOS as well? As if it is, then it might complicate the reference.

**Comment by gilgongo** [comment]
Is it the same as this dialogue? https://github.com/jamulussoftware/jamulus/issues/2041

**Comment by henkdegroot** [comment]
> Is it the same as this dialogue? #2041

Yes correct, that is the same dialog except it is for JACK instead of ASIO.
